revscoring/languages/dutch.py

```diff
@@ -42,62 +42,99 @@
 """
 
 badword_regexes = [
-    r"aars",
-    r"an(aal|us)\w*",
-    r"balhaar",
-    r"drol(len)?",
-    r"fack(en|ing|s)?", "facking",
-    r"flikkers?",
-    r"focking",
-    r"ge(ile?|lul)",
-    r"geneukt",
-    r"hoer(en?)?",
-    r"homos?",
-    r"kaka?",
-    r"kak(hoofd|ken)",
-    r"k[ae]nker",
-    r"klootzak(ken)?",
-    r"klote",
-    r"kont(gat|je)?",
-    r"pedo",
-    r"penis(sen)?",
-    r"peop",
-    r"piemels?",
-    r"pijpen",
-    r"pik",
-    r"pimel",
-    r"pipi",
-    r"poep(chinees?|en|hoofd)?",
-    r"poep(ie|je|sex|te?)s?",
-    r"porno?",
-    r"neuke?",
-    r"neuken(de)?",
-    r"neukt(en?)?",
-    r"stron(d|t)",
-    r"suck(s|t)?",
-    r"zuigt",
-    r"sukkels?",
-    r"ter(ing|ten)", "tetten",
-    r"tieten",
-    r"vagina",
-    r"verekte",
-    r"verkracht",
-    r"dikzak",
-    r"dildo",
-    r"mon?g(olen|ool)?", "mooiboy",
-    r"negers?",
-    r"shit",
-    r"sperma",
-    r"kut(jes?)?",
-    r"stelletje",
-    r"losers?",
-    r"lul(len)?",
-    r"reet",
-    r"scheet", "scheten", r"schijt",
-    r"diaree",
-    r"slet",
-    r"lekkerding",
-    r"likken"
+    # Curses
+    "aars",  # ass
+    "anaal", "anus",  # anal, anus
+    "balhaar",  # ball hair (testicular hair)
+    "debiel",  # infirm
+    "diaree", "diarree",  # diarrhea
+    "drol", "drollen",  # turd
+    "fack", "facking", "focking",  # misspelling of "fuck"
+    "flikker", "flikkers",  # perjorative for gay person ("faggot")
+    "geil", "geile",  # horny
+    "gelul",  # bullshit
+    "hoer", "hoere", "hoeren",  # whore
+    "homo", "homos",  # add "homo's" ; perjorative for gay person
```

Comment:
There's an unresolved TODO here for adding "homo's"

Reply:
Aha!  thanks.